### ISSUE TITLE
owslib lib - 'class' is not defined

### ISSUE THREAD

**Issue by palmerj** [microsoft/pyright#356]
**Describe the bug**
I get  'class' is not defined for some specific classes relating the owslib library

**To Reproduce**
Create simple script with:

```python
from owslib.iso import *

md = MD_Metadata(etree.parse("/my/path"))
```
Provides two errors:

_'MD_Metadata' is not definedpyright_

_'etree' is not definedpyright_

**Expected behavior**
The classes are correctly imported via import statement, so there should not be any errors.

**Comment by erictraut** [comment]
By default, pyright requires type stub files for imported libraries. Do you have a type stub available for owslib?

If you import symbols by name (`from owslib.iso import MD_Metadata, etree`), pyright will assign the imported symbols an "unknown" type if there is no type stub available. This will eliminate the errors you're seeing, but most advantages of type checking go away if types are unknown.

I recently added experimental support for inferring type information directly from library implementations. You can enable this through the VS Code setting "pyright.useLibraryCodeForTypes". Or if you're using the command-line version of pyright, you can pass "--lib" on the command line.

Note that inferring type information from the implementation is still a poor substitute for high-quality stub files that fully describe the types of all classes, methods, input parameters, return parameters, etc.

**Comment by palmerj** [comment]
OK thanks. Setting "pyright.useLibraryCodeForTypes" fixes the issue.
